code/datums/components/riding.dm

```diff
@@ -27,6 +27,15 @@
 	RegisterSignal(parent, COMSIG_MOVABLE_UNBUCKLE, .proc/vehicle_mob_unbuckle)
 	RegisterSignal(parent, COMSIG_MOVABLE_MOVED, .proc/vehicle_moved)
 
+/datum/component/riding/Destroy()
+	var/mob/M = parent
+	if(!istype(M))		//we absolutely can not runtime.
+		stack_trace("Warning: parent was not a human on [type]/Destroy().")
```

Comment:
When the parent is deleted, its components are deleted too. This will end up in useless stack traces every time a vehicle is deleted.